Backend/src/main/java/com/project/omega/bean/dto/ProductDTO.java

```diff
@@ -3,7 +3,6 @@
 import com.project.omega.bean.dao.entity.Supplier;
 
 public class ProductDTO {
-    private Long id;
```

Comment:
We're removing ids?

Reply:
Just from the DTO as we dont have control over it no need to change any models as any property that java doesnt find at the api end point will simply ignore